### DISCUSSION TITLE
String selection

### DISCUSSION THREAD

**Discussion by johnnywz00** [toerob/vscode-tads3tools#22] [Ideas]
This is just a small wish list item, but since this is the Ideas section, I thought I'd throw it out there.
The bottom line is a feature that with a key combo lets you select the entire string the cursor is in, rather than looking for word ends or line breaks. The thing is, sometimes I want to cut/paste the string with the quotes included, and sometimes I'm cutting the content out of the string (minus quotes, for instance to paste from a double-quote into a single-quote). Maybe too much to ask to have both versions!
I actually do this frequently enough that it would be a handy feature, so there's my idea!

**Comment by toerob** [comment]
I think this would be a nice addition and I might include it but I think we should investigate if there isn’t a plugin out there that already does. 

Anyhow, If it would become part of this plugin what shortcut key would be preferred as default?

**Reply by johnnywz00** [reply]
Could the key combo be user-assignable like the rest of VSCode keyboard settings?

**Reply by toerob** [reply]
Sure, that’s the idea, just thinking a _default_ combo would be nice to provide.

**Reply by johnnywz00** [reply]
Right... I don't know that I had one in mind...
